feat: add the coserv service to the native deployment (#424)
veraison/services: deployments/native/README.md

When doing this, you do not need to create/provide a root certificate.

### Step 4: set up signing keys

There are two signing keys: one is used to sign the EAR containing the
attestation results, and the other is used to sign the CoSERV query results.
The EAR key should be provided as `${VERAISON_ROOT}/signing/skey.jwk`.  The
CoSERV key should be provided as `${VERAISON_ROOT}/signing/coserv-signer.jwk`.

> Both keys must be in [JWK format](https://www.rfc-editor.org/rfc/rfc7517).

Reviewer: Why use different keys for CoSERV and attestation results? They're both results produced by the service, and have the same authority.

Reviewer 2: CoServ is a different service as compared to Veraison.

They have a different purpose and different role.

So it is good to keep the two separate services having two signing keys.

You can generate new keys for the deployment with

```bash
./deployment.sh init-signing-keys
```

Signing key generation relies on `step` which must be installed on the system
(see [dependencies](#dependencies) section above).

#### Using example signing key

> As with certificates, it is possible use the example keys instead of generating

Reviewer 2: ```suggestion
As with certificates, if possible use the example keys instead of generating
```

Author: I think this would change what is intended.
Here, we don't want to recommend using the example keys; we just want to point out that, rather than creating the credentials from scratch, one could use the pre-provided ones.
Ultimately, though, the choice is up to the user.
